BigData_updateCycling.R

```diff
@@ -29,12 +30,10 @@ PressSteadyDat <- PressSteadyDat %>%
 
 #   mutate(Config= replace(Config, Config == 'LRHL','HL'))  %>%
 
-# Read and summarize the sprint:
-PressSprintDat <- read.csv('C:/Users/eric.honert/Boa Technology Inc/PFL Team - General/Testing Segments/Cycling Performance Tests/CyclingDD_Jan2022/XSENSOR Data/TestDataSprintPressureData_DDJan22.csv')
 
-PressSprintDat <- PressSprintDat %>%
-  group_by(Subject, Config, Trial) %>%
-  summarize(HeelContact_sprint = mean(Sprint_HeelContactArea, na.rm = TRUE), PeakToePress_sprint = mean(PeakToePressure, na.rm = TRUE))
+PressSprintDat <- PressDat %>%
+  group_by(Subject, Config) %>%
```

Comment:
Do you need a filter statement here as well for the sprinting data?

Reply:
Yes I do, thanks! updated